packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -911,6 +916,27 @@ class ReactDOMServerRenderer {
           this.stack.push(frame);
           return '';
         }
+        case REACT_PLACEHOLDER_TYPE: {
+          if (enableSuspenseServerRenderer) {
```

Comment:
I'm not sure if it's worth adding this to the existing renderer, even in this limited form. If we do, we should emit a warning so people know to migrate to the new one.

Reply:
The idea is to only enable this flag in the FB build for now since we're already using placeholders.